backend/python/tapa/tapac.py

```diff
@@ -415,6 +429,8 @@ def main(argv: Optional[List[str]] = None):
       kwargs['user_max_usage_ratio'] = args.max_usage
     if args.force_dag is not None:
       kwargs['force_dag'] = args.force_dag
+    if args.floorplan_pre_assignments is not None:
+      kwargs['pre_assignments'] = args.floorplan_pre_assignments.name
```

Comment:
`args.floorplan_pre_assignments` is already a file object. Why do you pass the name instead of the object itself? Does autobridge need access to this file?